README.md

```diff
@@ -1,2 +1,8 @@
 # talks
 Talks, presentations and research papers published about Thoth
+
+# Blog posts
+
+ * [Use Kebechet machine learning to perform source code operations](https://developers.redhat.com/blog/2020/12/24/use-kebechet-machine-learning-to-perform-source-code-operations/)
+
+ * [AI software stack inspection with Thoth and TensorFlow](https://developers.redhat.com/blog/2020/09/30/ai-software-stack-inspection-with-thoth-and-tensorflow/)
```

Comment:
There is also: https://developers.redhat.com/blog/2019/10/28/microbenchmarks-for-ai-applications-using-red-hat-openshift-on-psi-in-project-thoth/?

Reply:
Added :+1: